frontend/src/__tests__/cypress/cypress/e2e/pipelines/PipelinesTopology.cy.ts

```diff
@@ -153,4 +210,64 @@ describe('Pipeline topology', () => {
         .contains('conditional-execution-pipeline');
     });
   });
+  describe('Pipelines logs', () => {
+    it('Test Logs tab', () => {
+      initIntercepts();
+      pipelineRunDetails.visit('test-project', 'test-pipeline-run-id');
+      pipelineRunDetails.findTaskNode('flip-coin').click();
+      pipelineRunDetails.findRightDrawer().findRightDrawerInputOutputTab().click();
+      pipelineRunDetails
+        .findRightDrawer()
+        .findRightDrawerDetailItem('output')
+        .findValue()
+        .contains('heads');
+      pipelineRunDetails.findRunTaskRightDrawer().findByText('Details').should('be.visible');
+      pipelineRunDetails.findRunTaskRightDrawer().findByText('Volumes').should('be.visible');
+      pipelineRunDetails.findRunTaskRightDrawer().findByText('Logs').should('be.visible');
+      pipelineRunDetails.findRightDrawer().findRightDrawerLogsTab().click();
```

Comment:
This is a long test, and I'm wondering if it could be broken up into multiple which would inherently add context to the operations taking place here. 

Either that, or at least, I think some brief comments above some of these operations would lend to a better understanding for others coming in to this file to update/debug this in the future.

Reply:
done. thanks.